src/lib/containers/synapse_form_wrapper/SynapseFormWrapper.stories.tsx

```diff
@@ -4,7 +4,7 @@ import SynapseFormWrapper from './SynapseFormWrapper'
 import { SynapseContextConsumer } from '../../utils/SynapseContext'
 
 export default {
-  title: 'Synapse/AccessTokenPage',
+  title: 'Synapse/SynapseFormWrapper',
```

Comment:
Wrong title caused these to collide. Discovered that SynapseFormWrapper isn't working in Storybook since we're missing some polyfills. Will create follow up ticket.

Reply:
https://sagebionetworks.jira.com/browse/SWC-6245